psi4/src/psi4/libcubeprop/csg.cc

```diff
@@ -695,4 +722,22 @@ std::pair<double, double> CubicScalarGrid::compute_isocontour_range(double* v2,
     }
     return std::make_pair(positive_isocontour, negative_isocontour);
 }
+std::string CubicScalarGrid::ecp_header() {
+    std::stringstream ecp_head;
+    ecp_head.str("");
+    if (primary_->has_ECP()) {
+        ecp_head << " Total core: " << primary_->n_ecp_core() << " [e] from 1-indexed atoms (";
+        std::stringstream ecp_atoms;
+        std::stringstream ecp_ncore;
+        for (int A = 0; A < mol_->natom(); A++) {
+            if (primary_->n_ecp_core(mol_->label(A)) > 0) {
+                ecp_atoms << A+1 << "[" << mol_->symbol(A).c_str() << "], ";
+                ecp_ncore << primary_->n_ecp_core(mol_->label(A)) << ", ";
```

Comment:
Nice & correct use of `label` vs. `symbol` here. c++ folks correct me if I'm wrong, but I don't think the `c_str()` is needed for `mol_->symbol(A).c_str()`.

Reply:
Correct.